test/test_loss.py

```diff
@@ -0,0 +1,82 @@
+"""Unit tests for the loss function."""
+from __future__ import print_function
+import json
+import numpy as np
+import tensorflow as tf
+from wavenet import make_loss, compute_receptive_field_size
+
+
+def make_logits():
```

Comment:
Instead of defining these functions, you could just create the tensors in the `setUp` method and save them as class attributes.